src/api/middleware.js

```diff
@@ -22,9 +22,15 @@ export function match(regexp: RegExp) {
   };
 }
 
-export function securityIframe(req: $RequestExtend, res: $ResponseExtend, next: $NextFunctionVer) {
+export function securityWebHeadersSecurity(req: $RequestExtend, res: $ResponseExtend, next: $NextFunctionVer) {
   // disable loading in frames (clickjacking, etc.)
-  res.header('X-Frame-Options', 'deny');
+  res.header(HEADERS.XFRAMES_OPTIONS, 'deny');
```

Comment:
Not sure if we should use Helmet https://expressjs.com/en/advanced/best-practice-security.html

Reply:
Actually I want to avoid a new dependency to maintain. Does it make sense ?

Reply:
Not sure, I have never used it directly.
It may be an option in the future but the current proposed solution should be sufficient.

Reply:
I think `helmet` is too big (https://bundlephobia.com/result?p=helmet@3.16.0) to achieve my goal. I'll skip it for now.

Reply:
What about https://www.npmjs.com/package/lusca?

Either way, these headers are set on the server, so won't change the bundle size.

Reply:
https://github.com/krakenjs/lusca/issues/127

We should keep it as is atm =)

Reply:
I wonder whether we need something else aside of the headers I added to this PR. Any thoughts?